apps/gateway/src/openApi/createApp.ts

```diff
@@ -0,0 +1,9 @@
+import { OpenAPIHono } from '@hono/zod-openapi'
+
+export function createRouter() {
+  return new OpenAPIHono({ strict: false })
```

Comment:
what does `strict` mean in there?

Reply:
Ni idea

Reply:
This https://hono.dev/docs/api/hono#strict-mode